code/game/objects/structures/xeno.dm

```diff
@@ -117,6 +117,9 @@
 		return FALSE
 
 	if(X.a_intent == INTENT_HARM) //Clear it out on hit; no need to double tap.
+		// No funny spitting and then removing thin spit to get free quick-builds
+		if(src.type == /obj/alien/resin/sticky && (SSmonitor.gamestate == SHUTTERS_CLOSED && (GLOB.master_mode == "Nuclear War" || GLOB.master_mode == "Distress Signal")))
```

Comment:
This is in violation of oop design and i do not get what it is doing anyway

Reply:
queen and hivelords have resin spit which leaves behind thin sticky resin , this could used to farm extra quickbuilds